frontend/src/tests/components/EarthquakesTest/EarthquakesTable.test.js

```diff
@@ -0,0 +1,88 @@
+import { fireEvent, render, waitFor } from '@testing-library/react';
+import { earthquakesFixtures } from 'fixtures/earthquakesFixtures';
+import EarthquakesTable from 'main/components/Earthquakes/EarthquakesTable';
+import { QueryClient, QueryClientProvider } from 'react-query';
+import { MemoryRouter } from 'react-router-dom';
+import { currentUserFixtures } from 'fixtures/currentUserFixtures';
+
+describe('EarthquakesTable tests', () => {
+  const queryClient = new QueryClient();
+
+  test('renders without crashing for empty table with user not logged in', () => {
+    const currentUser = null;
+
+    render(
+      <QueryClientProvider client={queryClient}>
+        <MemoryRouter>
+          <EarthquakesTable earthquakes={[]} currentUser={currentUser} />
+        </MemoryRouter>
+      </QueryClientProvider>
+    );
+  });
+
+  test('renders without crashing for empty table for ordinary user', () => {
+    const currentUser = currentUserFixtures.userOnly;
+
+    render(
+      <QueryClientProvider client={queryClient}>
+        <MemoryRouter>
+          <EarthquakesTable earthquakes={[]} currentUser={currentUser} />
+        </MemoryRouter>
+      </QueryClientProvider>
+    );
+  });
+
+  test('renders without crashing for empty table for admin', () => {
+    const currentUser = currentUserFixtures.adminUser;
+
+    render(
+      <QueryClientProvider client={queryClient}>
+        <MemoryRouter>
+          <EarthquakesTable earthquakes={[]} currentUser={currentUser} />
+        </MemoryRouter>
+      </QueryClientProvider>
+    );
+  });
+
+  test('Has the expected column headers and content for adminUser', () => {
+    const currentUser = currentUserFixtures.adminUser;
+
+    const { getByText, getByTestId } = render(
+      <QueryClientProvider client={queryClient}>
+        <MemoryRouter>
+          <EarthquakesTable
+            earthquakes={earthquakesFixtures.twoEarthquakes}
+            currentUser={currentUser}
+          />
+        </MemoryRouter>
+      </QueryClientProvider>
+    );
+
+    const expectedHeaders = ['id', 'Title', 'Magnitude', 'Place', 'Time'];
+    const expectedFields = ['id', 'title', 'mag', 'place', 'time'];
+    const testId = 'EarthquakesTable';
+
+    expectedHeaders.forEach((headerText) => {
+      const header = getByText(headerText);
+      expect(header).toBeInTheDocument();
+    });
+
+    expectedFields.forEach((field) => {
+      const header = getByTestId(`${testId}-cell-row-0-col-${field}`);
```

Comment:
This throws an error for one of the tests: "Unable to find an element by: [data-testid="EarthquakesTable-cell-row-0-col-title"]"

It seems like the data-testid is "EarthquakesTable-cell-row-0-col-Title" (with a capital T)
Not sure if that is the real issue tho, can't really think of anything else that might throw an error